geocoder/src/main/scala/grasshopper/geocoder/search/census/CensusGeocode.scala

```diff
@@ -8,7 +8,7 @@ import grasshopper.model.SearchableAddress
 import io.geojson.FeatureJsonProtocol._
 import org.elasticsearch.action.search.SearchType
 import org.elasticsearch.client.Client
-import org.elasticsearch.index.query.{ FilterBuilders, QueryBuilders }
+import org.elasticsearch.index.query.QueryBuilders
```

Comment:
`FilterBuilders` is now deprecated and everything is wrapped in `QueryBuilders`. Changes below should be consistent, but please review.